core/pta/attestation.c

```diff
@@ -272,8 +272,8 @@ static TEE_Result sec_storage_obj_write(TEE_UUID *uuid, uint32_t storage_id,
 	if (obj_id_len > TEE_OBJECT_ID_MAX_LEN)
 		return TEE_ERROR_BAD_PARAMETERS;
 
-	res = tee_pobj_get(uuid, (void *)obj_id, obj_id_len, flags, false,
-			   fops, &po);
+	res = tee_pobj_get(uuid, (void *)obj_id, obj_id_len, flags,
+			   TEE_POBJ_USAGE_OPEN, fops, &po);
```

Comment:
Should this be `TEE_POBJ_USAGE_OPEN` or `TEE_POBJ_USAGE_CREATE`?

Reply:
To maintain the previous behaviour it should be `TEE_POBJ_USAGE_OPEN` and that probably be good enough since this PTA isn't concurrent.

Reply:
Ok. I had the same concern for the StMM (now in the 1st commit of the series).
This P-R obviously aims at replacing 0/`false` with 0/`TEE_POBJ_USAGE_OPEN` but I think it's a good opportunity to challenge use of the later on write oprations.

Reply:
Using `TEE_POBJ_USAGE_CREATE` correctly might be a bit more trouble than it's worth. The main reason behind `tee_pobj_get()` is to enforce access logic as defined in the GP TEE Internal Core API spec. Here I guess it's mostly to avoid initializing `struct tee_pobj` directly like in `check_update_version()`.